kerl

```diff
@@ -162,10 +176,8 @@ get_releases()
 {
     if [ "$KERL_BUILD_BACKEND" = "git" ]
```

Comment:
Sure, but it needs to be outside this function as we redirect this output to file.